src/blog/2023/08/new-starter-tier.md

```diff
@@ -0,0 +1,60 @@
+---
+title: FlowFuse Cloud Starter package - The easiest way to get started with Node-RED
+subtitle: Introducing tiers and pricing changes for FlowFuse Cloud
+description: To allow the discovery of more features that FlowFuse has to offer FlowFuse is introducing a Starter package
+authors: ["nick-oleary"]
+image: blog/2023/08/images/ff-cloud-starter-package.png
+date: 2023-08-31
+tags:
+    - posts
+    - news
+    - flowfuse
+---
+
+FlowFuse is used by businesses and hobbyists alike. We aim to provide a platform that meets the needs of both, recognizing the importance of hobbyists in the Node-RED community and the long term success of FlowFuse as a company. We want to make it as easy as possible for people to get started on FlowFuse Cloud and help them understand the full value the platform offers. One of the barriers we've seen has been the current pricing structure of FlowFuse Cloud.
+
+To help more people understand the value FlowFuse offers we're launching a new Starter Package. For $15 per month you can run 2 Cloud instances and in addition manage 2 Node-RED instances on your own hardware with up to 2 team members. This provides much better value to get started on the platform.
+
+We are also introducing a new Team tier that allows for further growth beyond the limits of the Starter package.
+
+<!--more-->
```

Comment:
The teaser text is really long so it looks funny when it is on the blog page. I would suggest moving the <!--more--> to be after the first paragraph.